packages/content/src/Application/ContentWorkflow/Subscriber/UnpublishTransitionSubscriber.php

```diff
@@ -31,21 +31,15 @@
  */
 class UnpublishTransitionSubscriber implements EventSubscriberInterface
 {
-    /**
-     * @var DimensionContentRepositoryInterface
-     */
-    private $dimensionContentRepository;
-
     /**
      * @var EntityManagerInterface
      */
     protected $entityManager;
```

Comment:
This property was protected on an internal class that should not be extended. Now it's private.